src/solvers.jl

```diff
@@ -88,20 +120,31 @@ function solve(m::Model; suppress_warnings=false, ignore_solve_hook=(m.solvehook
         end
     end
 
+    # If the problem was solved, or if it terminated prematurely, try
+    # to extract a solution anyway. This commonly occurs when a time
+    # limit or tolerance is set (:UserLimit)
     if !(stat == :Infeasible || stat == :Unbounded)
         try
             objVal = MathProgBase.getobjval(m.internalModel) + m.obj.aff.constant
             colVal = MathProgBase.getsolution(m.internalModel)[1:numCols]
-            m.objVal = objVal # Don't corrupt the answers if one of the above two calls fails
+            # Don't corrupt the answers if one of the above two calls fails
+            m.objVal = objVal
             m.colVal = colVal
         end
     end
 
+    # The MathProgBase interface defines a conic problem to always be
+    # a minimization problem, so we need to flip the objective before
+    # reporting it to the user
     if traits.conic && m.objSense == :Max
         m.objVal *= -1
         scale!(m.linconstrDuals, -1)
```

Comment:
@mlubin this is a NOOP right because we don't support duals for conic problems, right?